lib/puppet/jenkins.rb

```diff
@@ -13,7 +17,11 @@ def self.home_dir
 
     # @return [String] Full path to the Jenkins user's plugin directory
     def self.plugins_dir
-      return File.join(self.home_dir, 'plugins')
+      if Facter.value(:osfamily) == 'OpenBSD'
+        return File.join(self.home_dir, '.jenkins', 'plugins')
```

Comment:
This is how the jenkins package works on OpenBSD, jenkins looks and picks up the plugins from ~_jenkins/.jenksins/plugins